### PR TITLE
(#4485) Wide Guide Card Line Height Fix

### LINKED ISSUE #4485: Fix Line-Height of WGC Heading
### Issue Description
The line height spacing on the Wide Guide Card (WGC) is incorrect. You can see this when the heading wraps one more than one line (see screenshot below).

### What's the expected change?
- The line-height of the Wide Guide Card Title should be token 2.

### What's the current functionality?
- The line-height of the Wide Guide Card Title is not token 2.

### Additional details / screenshot
- Image

### Related Tickets
* Original Story Ticket #4433

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #4485

### CODE REVIEW

**Comment by bennettcc** [comment]
PR should be on hold for review until the NCIDS release is merged: https://github.com/NCIOCPL/cgov-digital-platform/pull/4480

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/index.scss
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/index.scss:32]: The line-height should go into `typeset()` as the third option. 

Additionally, I wonder if the token for the line-height should be used (`$theme-heading-line-height`). The use of the token would be for the case in which the line-height of this heading should match the line height of prose. If that were the case, then this should probably use one of the heading tokens too (`$theme-h2-font-size`).

@andyvanavery31 - Is the intent that the font size of the Wide Guide Card should be the same as a visual/semantic H2 in prose? (Whereby if we globally changed the font-size of body content, this should change as well)

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
This is going to need to be rebased and probably fixed up after the NCIDS update/spacing changes. However, there is a change requested.

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/index.scss
Comment (andyvanavery31) [reply docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/index.scss:32]: @bryanpizzillo @karlikpj the intent is that the line-height for the WGC title is consistent with other headings of the same size, which is the same as nci-heading-h2 (font size: 24.96px and line-height: 29.952px). Yes it would be good to use a token for the font so that in the future if we made a change to global size it would apply to this.

**Review by olitharp-nci — APPROVED** [review]

**Review by andyvanavery31 — APPROVED** [review]
Passes Product Review.

**Review by bryanpizzillo — APPROVED** [review]

**Pull request merged**
